delete_product.php

```diff
@@ -0,0 +1,36 @@
+<?php
```

Comment:
The below code is really good where all the code is very clear focused with right spacing and comments